### DISCUSSION TITLE
Identify the segements of the mri scan possible?

### DISCUSSION THREAD

**Discussion by SirinC** [mlavik1/UnityVolumeRendering#257] [Q&A]
Is it possible to identify the segements of the mri scan with a learning process? Or is the the image quality not good enough to enable such segmentation?

**Comment by SirinC** [comment]
Hello, I would like to work with segment anything, but I don't know if the image quality is good enough to use this kind of learning method to identify the segments within the mri scan. Moreover I would like to know where I can find more MRI Scan Data to import it in a scene. I would really appreciate an answer to both of my questions. Thank you!

**Comment by mlavik1** [comment]
Hi! I'm sorry for the late reply, I was away during most of summer.

Overlay segmentations is on the TODO-list. There's actually a PR open for it that someone made before the summer, which I hope to get beck to soon. However, if you'd like to filter out parts of a volume I know Slicer has some toold/extensions for that. Apparently they also support Total Segmentator: https://discourse.slicer.org/t/new-extension-fully-automatic-whole-body-ct-segmentation-in-2-minutes-using-totalsegmentator/26710
I wonder if you could get somewhere with that?

Here's an open dataseg for segmented liver CTs: https://www.ircad.fr/research/data-sets/liver-segmentation-3d-ircadb-01/

And here's a list of open datasets that I wrote a while ago: https://github.com/mlavik1/UnityVolumeRendering/wiki/Sample-datasets

It looks like the Cancer Imaging Archive has some MRI datasets.

Hope that helped - if not please let me know! Have a nice day :)
